CHANGELOG.md

```diff
@@ -162,3 +162,10 @@ This release extends the full support for the Synergy and C7000 APIs to all the
 - Volume
 - Volume attachment
 - Volume template
+
+# Unreleased Changes
```

Comment:
This section should be on the first line, not last. The changelog order is: most recent to least recent